inc/languages/english/usercp.lang.php

```diff
@@ -140,7 +140,7 @@
 $l['email_notify_desc'] = "Selecting yes will automatically subscribe you to threads when you make a post in them.";
 $l['receive_pms'] = "Receive private messages from other users.";
 $l['receive_pms_desc'] = "Enables you to send and receive Private Messages.";
-$l['receive_from_buddy'] = "Only receive private messages from users on my <a href=\"#\" onclick=\"MyBB.popupWindow('/misc.php?action=buddypopup'); return false;\" target=\"_blank\">Buddy List</a>.";
+$l['receive_from_buddy'] = "Only receive private messages from users on my <a href=\"#\" onclick=\"MyBB.popupWindow('/misc.php?action=buddypopup'); return false;\" target=\"_blank\">Buddy List</a>. This setting has no effect unless there is at least one buddy on the list";
```

Comment:
Can you please add the missing period (`.`) on the end of this language string? Otherwise, this looks good.